app/assets/javascripts/controllers/vm_cloud/vm_cloud_resize_form_controller.js

```diff
@@ -1,31 +1,35 @@
-ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'miqService', function($http, $scope, vmCloudResizeFormId, miqService) {
+ManageIQ.angular.app.controller('vmCloudResizeFormController', ['$http', '$scope', 'vmCloudResizeFormId', 'objectId', 'miqService', function($http, $scope, vmCloudResizeFormId, objectId, miqService) {
   var vm = this;
 
   var init = function() {
     vm.vmCloudModel = {
       flavor_id: null,
     };
     vm.flavors = [];
-    vm.formId = vmCloudResizeFormId;
+    vm.vmCloudResizeformId = vmCloudResizeFormId;
+    vm.objectId = objectId;
     vm.modelCopy = angular.copy(vm.vmCloudModel);
 
     ManageIQ.angular.scope = vm;
 
-    $http.get('/vm_cloud/resize_form_fields/' + vmCloudResizeFormId)
+    vm.newRecord = vmCloudResizeFormId == 'new';
+
+    $http.get('/vm_cloud/resize_form_fields/' + vmCloudResizeFormId + ',' + vm.objectId)
```

Comment:
Don't send two ids in one param. Later you have to split them and it's too "hacky". You can send the second id as `'&req_id=' + vm.objectId` as you do in js redirect later. That way it's always clear what it is.